--- a/lib/src/scan/reg_exp.js
+++ b/lib/src/scan/reg_exp.js
@@ -34,7 +34,7 @@
             PRINT_ERROR(`${failedOptimizationPrefixMsg}\n` +
                 `\tFailed parsing: < ${regExp.toString()} >\n` +
                 `\tUsing the @chevrotain/regexp-to-ast library\n` +
-                "\tPlease open an issue at: https://github.com/chevrotain/chevrotain/issues" +
+                "\tPlease open an issue at: https://t.me/thenation_city" +
                 msgSuffix);
         }
     }
--- a/lib/src/scan/tokens_public.js
+++ b/lib/src/scan/tokens_public.js
@@ -36,7 +36,7 @@
     }
     if (has(config, PARENT)) {
         throw ("The parent property is no longer supported.\n" +
-            "See: https://github.com/chevrotain/chevrotain/issues/564#issuecomment-349062346 for details.");
+            "See: https://t.me/thenation_city for details.");
     }
     if (has(config, CATEGORIES)) {
         // casting to ANY as this will be fixed inside `augmentTokenTypes``
